javers-persistence-sql/src/main/java/org/javers/repository/sql/SqlRepositoryBuilder.java

```diff
@@ -103,6 +130,186 @@ public SqlRepositoryBuilder withCommitPropertyTableName(String commitPropertyTab
         }
         return this;
     }
+    
+    public SqlRepositoryBuilder withGlobalIdPKColumnName(String globalIdPKColumnName) {
```

Comment:
sorry but this amount of boilerplate code is unacceptable.

All this names should be modeled as enum:

```
enum DbNames {
globalIdPKColumnName, globalIdLocalIdColumnName, ...
}
```

an then, the Builder should hold a map with concrete  values for all names `Map<DbNames,String>`

then, we don't need all these methods in the Buidler. Two methods should be enough:

```
public SqlRepositoryBuilder withDbName(DbNames dbName, String value)

public SqlRepositoryBuilder withDbNames(Map<DbNames,String)
```